src/data/CassandraBackend.hpp

```diff
@@ -567,6 +567,32 @@ class BasicCassandraBackend : public BackendInterface {
         return std::nullopt;
     }
 
+    std::vector<Blob>
```

Comment:
You should try to return a pair of <uint32_t, Blob> instead, since you would want to know the sequence of each blob. You would need to change on line 582